opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -67,6 +67,7 @@
 #include <opm/parser/eclipse/EclipseState/Tables/SwofTable.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableContainer.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/WatvisctTable.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/DensityTable.hpp>
```

Comment:
> is DENSITY a table keyword?

Good question; I guess the `DENSITY` keyword goes like a thermodynamic property - which is what you will find in the tables directory. It is a border-line case.